app/commands/v2/put_content.rb

```diff
@@ -200,6 +202,29 @@ def clear_draft_item_of_different_locale_but_matching_base_path
         )
       end
 
+      def update_draft_items_of_different_locale_but_matching_content_id
+        return unless payload[:base_path]
+        return if Document.where(content_id: payload[:content_id], locale: payload[:locale]).any?
```

Comment:
Is it worth adding a couple of tests for these early returns? Might also help serve as documentation.

Reply:
Yes, that's a good idea, I'll add them.